app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java

```diff
@@ -159,6 +163,18 @@ private void buyTicket() {
         startActivity(intent);
     }
 
+    private void addToCalendar() {
+        Intent intent = new Intent(Intent.ACTION_INSERT)
+                .setData(CalendarContract.Events.CONTENT_URI)
+                .putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, event.getDate().getTime())
+                .putExtra(CalendarContract.EXTRA_EVENT_END_TIME, event.getDate().getTime() + 7200000)
```

Comment:
We currently do not have a duration for events.
Assuming two hours for all events should only be a temporary workaround.
Maybe add a TODO to adjust this when/if a model adjustment of the event class is possible.

@kordianbruck @schulzalexander Is it possible and does it make sense to provide a duration for events?

Reply:
We talked about it shortly in the Hangouts meeting today. We agreed that it is not a critical issue at the moment. But if we think this might be a reasonable addition in the future, we could add a ticket now, so that we do not forget it.

Reply:
Nice. I keep it now